.github/workflows/CI.yml

```diff
@@ -119,7 +119,7 @@ jobs:
         shell: bash
       - name: Setup node x86
         if: matrix.settings.target == 'i686-pc-windows-msvc'
-        run: pnpx @ialdama/jsonmod --key pnpm.supportedArchitectures.cpu --values ia32
+        run: pnpx @ialdama/jsonmod --key pnpm.supportedArchitectures.cpu --values ia32 current
```

Comment:
interesting. I did not know about this tool.

Reply:
I just created it 😅